iotfunctions/stages.py

```diff
@@ -631,20 +647,23 @@ def _push_alert_events_to_dd(self, all_alert_events, index_has_entity_id):
 
                 # Setup alert event for Data Dictionary
                 timestamp_in_nano_seconds = int(tmp_timestamp.to_datetime64())
-                timestamp_string = f"{tmp_timestamp.year:04}-{tmp_timestamp.month:02}-{tmp_timestamp.day:02} " \
-                                   f"{tmp_timestamp.hour:02}:{tmp_timestamp.minute:02}:{tmp_timestamp.second:02}." \
-                                   f"{tmp_timestamp.microsecond:06}{tmp_timestamp.nanosecond:03}"
+                #timestamp_string = _timestamp_as_string(tmp_timestamp)
+                created_ts_in_nano_seconds = int(event_row[2].to_datetime64())
+                #created_ts_string = _timestamp_as_string(event_row[2])
                 alert_attributes = {"alertId": event_row[1],
                                     "name": alert_name,
-                                    "timestamp": timestamp_string,
-                                    "owner": None,
-                                    "severity": cached_values[0],
-                                    "priority": cached_values[1],
-                                    "status": cached_values[2],
-                                    "metadata": None,
-                                    "createdTs": None,
-                                    "updatedTs": None,
-                                    "lastUpdatedBy": None}
+                                    "timestamp": timestamp_in_nano_seconds,
+                                    "resourceDDId": self.dms.entity_type_dd_id,
+                                    "createdTs": created_ts_in_nano_seconds,
+                                    "updatedTs": created_ts_in_nano_seconds}
```

Comment:
Ideally, these values should come from the database. 
Create an entry in alert in the database
then fetch the newly created/updated alerts from the database and then push to DD.

Reply:
Actually, they do! By the way, pipeline set created_ts = updated_ts when pushing to db. Therefore I only retrieve created_ts from database.